cmd/server/main.go

```diff
@@ -1,15 +1,32 @@
 package main
 
 import (
+	"os/exec"
+	"strings"
+
 	"go.keploy.io/server/server"
 )
 
 // version is the version of the server and will be injected during build by ldflags
 // see https://goreleaser.com/customization/build/
 
-var version = server.DefaultVersion
+var version string
 
 func main() {
 	// main method to start Keploy server
+	if version == "" {
+		version = getKeployVersion()
+	}
 	server.Server(version)
 }
+
+func getKeployVersion() string {
+	cmd := exec.Command("git", "describe", "--tags")
+	out, err := cmd.Output()
+	if err != nil {
+		return "0.1.0-dev"
+	}
+	version := strings.TrimSpace(string(out))
+	version = version[0:6] + "-dev"
```

Comment:
This would cause segmentation fault if something goes wrong, instead of seg fault, we can add graceful failure handling (like only accessing `version[0:6]` when the length is atleast "x". In other cases, we default to some value

Reply:
This issue will get resolved as go-get is used now.